SocialFilter/sections/specific.txt

```diff
@@ -46,6 +46,7 @@ gpmradio.ru,originalnews.nico,archives.gov,sportnews.to,nationalchronicle.in,ana
 ! ##.sns_area
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
+orient-anime.jp##.l-footer__in.-share 
```

Comment:
`orient-anime.jp##.-share ` is enough

Reply:
ok. thank you for advice.
I dont have to update it?

Reply:
Yes, I changed the rule